src/components/chips/js/contactChipsDirective.js

```diff
@@ -63,6 +65,8 @@ var MD_CONTACT_CHIPS_TEMPLATE = '\
               md-items="item in $mdContactChipsCtrl.queryContact($mdContactChipsCtrl.searchText)"\
               md-item-text="$mdContactChipsCtrl.itemName(item)"\
               md-no-cache="true"\
+              md-min-length="$mdContactChipsCtrl.minLength"\
+              md-has-not-found="true"\
```

Comment:
@topherfangio while doing further research I found a bug on the master branch that isn't in the v1.1.0-rc5 release. When I remove the `md-not-found` element in the basic `md-autocomplete` example I would expect that the search function would still work since the `md-min-length` equals 0. This is however not the case on anymore on the master branch while seems to work in the v1.1.0-rc5 release.

I'm doing some further research.